sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,772 @@
+  align with the SYCL 2020 spec changes. intel/llvm#13238 intel/llvm#13893
+- Added support for 1- and 2-byte data types to ESIMD prefetch APIs.
+  intel/llvm#13452
+- Enabled `ext_intel_matrix` support for Intel GNR devices. intel/llvm#14436
+- Added support for 1x64x16 `bfloat16` matrices on PVC> intel/llvm#13391
+- Added new overloads of `load_2d`, `store_2d` or `prefetch_2d` ESIMD APIs that
+  accept compile-time properties. intel/llvm#13046
+- Added support for `shift_group_left`, `shift_group_right`,
+  `permute_group_by_xor` and `select_from_group` algorithms for non-uniform
+  groups. intel/llvm#12705
+- Removed Plugin Interface. That is a collection of internal libraries which
+  implemented unified interface to various lower-level runtimes like OpenCL,
+  Level Zero, etc. It is now completely replaced by
+  [Unified Runtime](https://github.com/oneapi-src/unified-runtime/) and this
+  removal should reduce amount and size of redistributable libraries.
+- Enhanced ESIMD `slm_atomic_update` API to also support `fsub` and `fadd`
+  operations. intel/llvm#13535
+- Lifted some of the restrictions from ESIMD `block_store` API. intel/llvm#13150
+- Added support for graph update functionality. intel/llvm#12840
+- Added support for external semaphore wait and signal operations that can take
+  a value to bindless images extension. intel/llvm#13860
+- Added support for device-to-device copying of `image_device_handle`.
+  intel/llvm#12449
+- Improved performance of `queue::fill` on CUDA backend by making it use 2- and
+  4-byte operations instead of only using 1-byte operations. intel/llvm#13788
+  intel/llvm#13779
+- Enhanced `sycl_ext_oneapi_graph` extension implementation on Level Zero
+  backend by taking advantage of copy engines available on some devices to be
+  able to execute kernels and memory operations in parallel. intel/llvm#13051
+- Expanded list of supported atomic memory fence scopes supported by HIP
+  devices. intel/llvm#12872
+- Added SYCL wrappers for more Intel Math Functions. SYCL wrappers are in
+  `sycl::ext::intel::math::` namespace and provide nicer names that do not
+  start with `__`. intel/llvm#13762
+- Added initial support for bindless images on Intel GPUs (only DG2 and MTL).
+  It is very limited in functionality that is supported and it only works if
+  certain environment variables are set. Therefore, it is not yet ready for a
+  wide adoption, but could be used for very early adoption. intel/llvm#13946
+  intel/llvm#13753 intel/llvm#14493 intel/llvm#14266
+- Introduced an optimization to graphs where linear graphs would require less
+  amount of synchronization. This optimization is backend-specific and only
+  performed in very limited amount of cases. intel/llvm#13088
+- Implemented new `fetch_image` overload which accepts sampled image and
+  coordinates. intel/llvm#12447
+- Extended address sanitizer support to cover Intel GPU devices besides CPU
+  devices. intel/llvm#13450
```

Comment:
The PR #13450 is to support device sanitizer on DG2 GPU device. We have supported CPU and PVC GPU devices before this PR. Looks like this description is not accurate.

Reply:
Thanks, applied in c77b5d3a8c25b88c9fd20d881c49e46073765f85